clang/test/Headers/lasxintrin.c

```diff
@@ -0,0 +1,6 @@
+// REQUIRES: loongarch-registered-target
+// RUN: %clang_cc1 %s -fsyntax-only -triple loongarch64-linux -target-feature +lasx
```

Comment:
Should we emphasize linux?

Reply:
-linux dropped.  I just mistakenly thought a triple had to have the os field.